### PR TITLE
Numerous CSS tweaks

### PR DESCRIPTION
**Pull request by petey**
A quick pass with Jay through a few of our pages to tweak font sizes and colors.

screen shot 2016-09-22 at 5 46 09 pm

screen shot 2016-09-22 at 5 46 20 pm

### CODE REVIEW

**Comment by petey** [comment]
screen shot 2016-09-22 at 6 01 05 pm

File: app/components/home-hero/template.hbs [hunk 1]
```diff
@@ -4,65 +4,73 @@
...
+    </div>
+    <div class="cell pure-u-1-2">
+      <div>
+        <h2>Runs Anywhere</h2>
+        <p>
+          Screwdriver's architecture uses pluggable components under the hood
+          to allow you to swap out the pieces that make sense for your infrastructure.
+          Swap in Postgres for the Datastore or Jenkins for the Executor.
+          You can even dynamically select an execution engine based on the needs of each pipeline.
+          For example, send golang builds to the kubernetes executor while your iOS builds got to a
```
Comment (tkyi) [inline app/components/home-hero/template.hbs]: For example, send **Golang** builds to **a Kubernetes** executor while your iOS builds **go** to a Jenkins execution farm.

Comment (petey) [inline app/components/home-hero/template.hbs]: I think we should update the copy in another PR. There are a number of things that aren't entirely accurate.

Comment (tkyi) [inline app/components/home-hero/template.hbs]: 👍 sounds good

**Pull request merged**